taxi_service/settings.py

```diff
@@ -20,8 +21,7 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-8ovil3xu6=eaoqd#" \
-             "-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
+SECRET_KEY = os.getenv("DJANGO_SECRET_KEY", "fallback-secret-key")
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
The `DEBUG` setting is currently set to `True`. Remember to set it to `False` in a production environment to avoid exposing sensitive information.